src/Microsoft.HttpRepl/Commands/PrefCommand.cs

```diff
@@ -177,6 +178,19 @@ private void SetSetting(IShellState shellState, HttpState programState, DefaultC
             {
                 shellState.ConsoleManager.Error.WriteLine(Resources.Strings.PrefCommand_Error_Saving.SetColor(programState.ErrorColor));
             }
+            else
+            {
+                // If we think they're configurating HttpRepl to use Visual Studio Code as their editor, we should
+                // warn them that for best integration, they should also pass the `-w` or `--wait` arguments to
+                // Visual Studio Code.
+                if (string.Equals(prefName, WellKnownPreference.DefaultEditorCommand, StringComparison.Ordinal))
+                {
+                    if (IsVSCode(prefValue))
```

Comment:
Should/can we also check if they have already set the argument correctly?  E.g. if I'm switching from Code to Code-Insiders, maybe I don't need the warning.  Or if I set the arguments before the editor.  Showing the warning will make me double-check it, and possibly waste my time.  (This is a pretty nitty case though)

Reply:
For the moment, I'm going to say that inconvenience is small enough that it doesn't warrant trying to parse someone's existing arguments (especially since `-w` could be inside another argument (e.g. `--new-window`) or a quoted argument (no e.g. off the top of my head, but we'd want to avoid it if we were actually parsing)).